src/test/java/cz/tomasdvorak/eet/client/security/MerlinWithCRLDistributionPointsExtensionTest.java

```diff
@@ -29,16 +30,16 @@ public void setUp() throws Exception {
         keystore = getTruststore("/certificates/rca15_rsa.der", "/certificates/2qca16_rsa.der");
     }
 
-
+    @Ignore("Disabled due to expired test certificate as well")
```

Comment:
I put a maximum effort to provide a valid certificate, but I failed - stopped on getting `No message with ID "certpath" found in resource bundle`